.gitignore

```diff
@@ -1,3 +1,4 @@
+/stack.yaml
```

Comment:
This shouldn't be included in a template, we'll want the users of the template to check in their `stack.yaml` file.